TOC.md

```diff
@@ -2,187 +2,209 @@
+  + Get Started
+    + Start a Local Cluster
+      - [From Binary](op-guide/binary-local-deployment.md)
+      - [In Kubernetes](op-guide/kubernetes-local.md)
+      - [In Docker Compose](op-guide/docker-compose.md)
+    - [Explore SQL with TiDB](try-tidb.md)
+    - [Import Sample Database](bikeshare-example-database.md)
+    - [Read Historical Data](op-guide/history-read.md)
+  - Deploy
+    - [Hardware Recommendations](op-guide/recommendation.md)
+    + From Binary Tarball
+      - [For testing environments](op-guide/binary-testing-deployment.md)
+      - [For production environments](op-guide/binary-deployment.md)
+    + Orchestrated Deployment
+      - [Ansible Deployment (Recommended)](op-guide/ansible-deployment.md)
+      - [Ansible Offline Deployment](op-guide/offline-ansible-deployment.md)
+      - [Docker Deployment](op-guide/docker-deployment.md)
+      - [Kubernetes Deployment](op-guide/kubernetes.md)
+      - [Overview of Ansible Operations](op-guide/ansible-operation.md)
+    + Geographic Redundancy
+      - [Overview](op-guide/cross-dc-deployment.md)
+      - [Configure Location Awareness](op-guide/location-awareness.md)
+    - [TiSpark](tispark/tispark-quick-start-guide.md)
+    - [Data Migration with Ansible](tools/dm/deployment.md)
+  + Secure
+    - [Security Compatibility with MySQL](sql/security-compatibility.md)
+    - [The TiDB Access Privilege System](sql/privilege.md)
+    - [TiDB User Account Management](sql/user-account-management.md)
+    + Transport Layer Security (TLS)
+      - [Enable TLS For MySQL Clients](sql/encrypted-connections.md)
+      - [Enable TLS Between TiDB Components](op-guide/security.md)
     - [Generate Self-signed Certificates](op-guide/generate-self-signed-certificates.md)
-    - [Cluster Topology Configuration](op-guide/location-awareness.md)
   + Monitor
-    - [Monitoring Framework Overview](op-guide/monitor-overview.md)
+    - [Overview](op-guide/monitor-overview.md)
     - [Monitor a TiDB Cluster](op-guide/monitor.md)
     + Key Monitoring Metrics
       - [Overview](op-guide/dashboard-overview-info.md)
       - [TiDB](op-guide/tidb-dashboard-info.md)
       - [PD](op-guide/dashboard-pd-info.md)
       - [TiKV](op-guide/dashboard-tikv-info.md)
-  + Scale
-    - [Scale a TiDB Cluster](op-guide/horizontal-scale.md)
-    - [Scale Using Ansible](op-guide/ansible-deployment-scale.md)
-  + Upgrade
-    - [Upgrade the Component Version](op-guide/ansible-deployment-rolling-update.md#upgrade-the-component-version)
-    - [TiDB 2.0 Upgrade Guide](op-guide/tidb-v2.0-upgrade-guide.md)
-    - [TiDB 2.1 Upgrade Guide](op-guide/tidb-v2.1-upgrade-guide.md)
-    - [FAQs After Upgrade](op-guide/upgrade-faq.md)
-  - [Tune Performance](op-guide/tune-tikv.md)
-  + Backup and Migrate
+  + Migrate
+    - [Overview](op-guide/migration-overview.md)
+    - [From MySQL](op-guide/migration.md)
+    - [Using Data Migration](tools/dm/overview.md)
+  + Maintain
     - [Backup and Restore](op-guide/backup-restore.md)
```

Comment:
Missed incremental import using Syncer.

Reply:
Suggest re-arranging as:
Overview
Full import using mydumper/loader
Incremental import using Syncer. 
(need to split the current backup-restore.md file)

Reply:
I believe you mean split `op-guide/migration.md`, which I agree with you.  This is an improvement: https://github.com/pingcap/docs/pull/1037